### DISCUSSION TITLE
Cache imported remote schema locally

### DISCUSSION THREAD

**Discussion by qw-in** [dotansimha/graphql-code-generator#5300] [Ideas] [answered]
Loading a remote schema can be expensive & slow in some cases. Of course this can be avoided by keeping a local copy but that then needs to be checked into VC & kept up to date.

I am wondering if there is interest in a local cache? For example the schema is imported as normal and then cached under `node_modules/.cache/@graphql-codegen` or similar.

**Comment by ardatan** [comment, marked as answer]
You can fetch the schema as an SDL with schema-ast then use it
